### PR TITLE
Rename identical package slugs and names

### PR DESCRIPTION
**Pull request by mbenguig**

### CODE REVIEW

**Review by medou-boushab — APPROVED** [review]

File: DataConnectors/resources/catalog/Cassandra.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <job
   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xmlns="urn:proactive:jobdescriptor:3.14" xsi:schemaLocation="urn:proactive:jobdescriptor:3.14 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.14/schedulerjob.xsd"  name="Cassandra" tags="Cloud Storage,Data Connectors,NoSQL,Cassandra,Data Warehouse,SQL" projectName="3. NoSQL" priority="normal" onTaskError="continueJobExecution"  maxNumberOfExecution="2" >
+  xmlns="urn:proactive:jobdescriptor:3.14" xsi:schemaLocation="urn:proactive:jobdescriptor:3.14 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.14/schedulerjob.xsd"  name="Cassandra" tags="Data Connectors,Cassandra,Cloud Storage,Data Warehouse,NoSQL" projectName="3. NoSQL" priority="normal" onTaskError="continueJobExecution"  maxNumberOfExecution="2" >
```
Comment (fviale) [inline DataConnectors/resources/catalog/Cassandra.xml:4]: why associate Cassandra with Cloud Storage ?

File: DataConnectors/resources/catalog/Database_Sequential_Queries.xml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <job
   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xmlns="urn:proactive:jobdescriptor:3.14" xsi:schemaLocation="urn:proactive:jobdescriptor:3.14 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.14/schedulerjob.xsd"  name="Database_Sequential_Queries" tags="Cloud Storage,Data Connectors,NoSQL,Data Warehouse,SQL" projectName="2. SQL" priority="normal" onTaskError="continueJobExecution"  maxNumberOfExecution="2"  >
+  xmlns="urn:proactive:jobdescriptor:3.14" xsi:schemaLocation="urn:proactive:jobdescriptor:3.14 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.14/schedulerjob.xsd"  name="Database_Sequential_Queries" tags="Data Connectors,NoSQL,SQL" projectName="2. SQL" priority="normal" onTaskError="continueJobExecution"  maxNumberOfExecution="2"  >
```
Comment (fviale) [inline DataConnectors/resources/catalog/Database_Sequential_Queries.xml:4]: all the supported databases (see below) are SQL dbs, so NoSQL tag should be removed

Comment (fviale) [reply DataConnectors/resources/catalog/Database_Sequential_Queries.xml:4]: you could also add the tags of each supported db

File: DataConnectors/resources/catalog/Process_Rows_Automatically.xml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <job
   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xmlns="urn:proactive:jobdescriptor:3.14" xsi:schemaLocation="urn:proactive:jobdescriptor:3.14 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.14/schedulerjob.xsd"  name="Process_Rows_Automatically" tags="Cloud Storage,Data Connectors,NoSQL,Data Warehouse,SQL" projectName="2. SQL" priority="normal" onTaskError="continueJobExecution"  maxNumberOfExecution="2"  >
+  xmlns="urn:proactive:jobdescriptor:3.14" xsi:schemaLocation="urn:proactive:jobdescriptor:3.14 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.14/schedulerjob.xsd"  name="Process_Rows_Automatically" tags="Data Connectors,SQL" projectName="2. SQL" priority="normal" onTaskError="continueJobExecution"  maxNumberOfExecution="2"  >
```
Comment (fviale) [inline DataConnectors/resources/catalog/Process_Rows_Automatically.xml:4]: you could also add the tags of each supported db

File: DataConnectors/resources/catalog/SQL_Pooled_Connection_Query.xml [hunk 4]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <job
   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xmlns="urn:proactive:jobdescriptor:3.14" xsi:schemaLocation="urn:proactive:jobdescriptor:3.14 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.14/schedulerjob.xsd"  name="SQL_Pooled_Connection_Query" tags="Cloud Storage,Data Connectors,NoSQL,Data Warehouse,SQL" projectName="2. SQL" priority="normal" onTaskError="continueJobExecution"  maxNumberOfExecution="2" >
+  xmlns="urn:proactive:jobdescriptor:3.14" xsi:schemaLocation="urn:proactive:jobdescriptor:3.14 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.14/schedulerjob.xsd"  name="SQL_Pooled_Connection_Query" tags="Data Connectors,SQL" projectName="2. SQL" priority="normal" onTaskError="continueJobExecution"  maxNumberOfExecution="2" >
```
Comment (fviale) [inline DataConnectors/resources/catalog/SQL_Pooled_Connection_Query.xml:4]: you could also add the tags of each supported db

File: DataConnectors/resources/catalog/SQL_Pooled_Connection_Update.xml [hunk 5]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <job
   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xmlns="urn:proactive:jobdescriptor:3.14" xsi:schemaLocation="urn:proactive:jobdescriptor:3.14 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.14/schedulerjob.xsd"  name="SQL_Pooled_Connection_Update" tags="Cloud Storage,Data Connectors,NoSQL,Data Warehouse,SQL" projectName="2. SQL" priority="normal" onTaskError="continueJobExecution"  maxNumberOfExecution="2" >
+  xmlns="urn:proactive:jobdescriptor:3.14" xsi:schemaLocation="urn:proactive:jobdescriptor:3.14 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.14/schedulerjob.xsd"  name="SQL_Pooled_Connection_Update" tags="Data Connectors,SQL" projectName="2. SQL" priority="normal" onTaskError="continueJobExecution"  maxNumberOfExecution="2" >
```
Comment (fviale) [inline DataConnectors/resources/catalog/SQL_Pooled_Connection_Update.xml:4]: you could also add the tags of each supported db

File: DataConnectors/resources/catalog/SQL_Server.xml [hunk 6]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <job
   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xmlns="urn:proactive:jobdescriptor:3.14" xsi:schemaLocation="urn:proactive:jobdescriptor:3.14 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.14/schedulerjob.xsd"  name="SQL_Server" tags="Cloud Storage,Data Connectors,NoSQL,Data Warehouse,SQL Server,SQL" projectName="2. SQL" priority="normal" onTaskError="continueJobExecution"  maxNumberOfExecution="2" >
+  xmlns="urn:proactive:jobdescriptor:3.14" xsi:schemaLocation="urn:proactive:jobdescriptor:3.14 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.14/schedulerjob.xsd"  name="SQL_Server" tags="Data Connectors,SQL" projectName="2. SQL" priority="normal" onTaskError="continueJobExecution"  maxNumberOfExecution="2" >
```
Comment (fviale) [inline DataConnectors/resources/catalog/SQL_Server.xml:4]: please add the tag "SQL Server"

File: DataConnectors/resources/catalog/SQL_Statements.xml [hunk 7]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <job
   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xmlns="urn:proactive:jobdescriptor:3.14" xsi:schemaLocation="urn:proactive:jobdescriptor:3.14 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.14/schedulerjob.xsd"  name="SQL_Statements" tags="Cloud Storage,Data Connectors,NoSQL,Data Warehouse,SQL" projectName="2. SQL" priority="normal" onTaskError="continueJobExecution"  maxNumberOfExecution="2" >
+  xmlns="urn:proactive:jobdescriptor:3.14" xsi:schemaLocation="urn:proactive:jobdescriptor:3.14 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.14/schedulerjob.xsd"  name="SQL_Statements" tags="Data Connectors,SQL" projectName="2. SQL" priority="normal" onTaskError="continueJobExecution"  maxNumberOfExecution="2" >
```
Comment (fviale) [inline DataConnectors/resources/catalog/SQL_Statements.xml:4]: you could also add the tags of each supported db

File: DataConnectors/resources/catalog/Process_Rows_Automatically.xml (hunk 3 above)
Comment (mbenguig) [reply DataConnectors/resources/catalog/Process_Rows_Automatically.xml:4]: Ok

File: DataConnectors/resources/catalog/SQL_Server.xml (hunk 6 above)
Comment (mbenguig) [reply DataConnectors/resources/catalog/SQL_Server.xml:4]: Ok

File: DataConnectors/resources/catalog/SQL_Statements.xml (hunk 7 above)
Comment (mbenguig) [reply DataConnectors/resources/catalog/SQL_Statements.xml:4]: Ok

**Pull request merged**